graphql/config_resolver.go

```diff
@@ -32,3 +32,17 @@ func (r *Resolver) SpruceConfig() SpruceConfigResolver { return &spruceConfigRes
 
 type containerPoolResolver struct{ *Resolver }
 type spruceConfigResolver struct{ *Resolver }
+
+// !!! WARNING !!!
+// The code below was going to be deleted when updating resolvers. It has been copied here so you have
+// one last chance to move it out of harms way if you want. There are two reasons this happens:
+//  - When renaming or deleting a resolver the old code will be put in here. You can safely delete
+//    it when you're done.
+//  - You have helper methods in this file. Move them out to keep these resolver files clean.
+/*
+	func (r *adminSettingsResolver) Announcements(ctx context.Context, obj *model.APIAdminSettings) (*Announcements, error) {
+	panic(fmt.Errorf("not implemented: Announcements - announcements"))
+}
+func (r *Resolver) AdminSettings() AdminSettingsResolver { return &adminSettingsResolver{r} }
+type adminSettingsResolver struct{ *Resolver }
+*/
```

Comment:
You can manually delete this block, then run `make gqlgen` to make sure it doesn't come back